packages/baggage-span-processor/test/baggage-span-processor.test.ts

```diff
@@ -131,17 +120,14 @@ describe('BaggageSpanProcessor with regex key filter', () => {
   let span: Span;
 
   beforeEach(() => {
-    span = new Span(
-      new BasicTracerProvider().getTracer('baggage-testing'),
-      ROOT_CONTEXT,
+    const tracer = new BasicTracerProvider().getTracer('baggage-testing');
+    span = tracer.startSpan(
       'Edward W. Span',
       {
-        traceId: 'e4cda95b652f4a1592b449d5929fda1b',
-        spanId: '7e0c63257de34c92',
-        traceFlags: TraceFlags.SAMPLED,
+        kind: SpanKind.SERVER,
       },
-      SpanKind.SERVER
-    );
+      ROOT_CONTEXT
+    ) as unknown as Span;
```

Comment:
Do we need both type coercions here - can we coerce straight into `Span`?

Reply:
Not necessary at all. Removing it in the next commit. Good catch :)

Reply:
Removed!